docker_images/fasttext/app/main.py

```diff
@@ -39,6 +39,7 @@
 ALLOWED_TASKS: Dict[str, Type[Pipeline]] = {
     "feature-extraction": FeatureExtractionPipeline,
     "text-classification": TextClassificationPipeline,
+    "language-identification": TextClassificationPipeline,
```

Comment:
`language-identification` is not a separate task, it's a subtask of `text-classification` (see https://github.com/huggingface/hub-docs/blob/main/js/src/lib/interfaces/Types.ts). This code can be removed. All repos for `language-identification` also need the `text-classification` tag so they get the right inference API path as well as the widget.